controllers/v1/programs.js

```diff
@@ -3,13 +3,385 @@
  * author : vishnu
  * created-date : 09-Mar-2022
  * Description : programs related information.
- */
+*/
+
+
+// Dependencies
+const programsHelper = require(MODULES_BASE_PATH + "/programs/helper")
+
+
 module.exports = class  Programs extends Abstract{
-    constructor() {
-      super("programs");
+  constructor() {
+    super("programs");
+  }
+  static get name() {
+    return "programs";
+  }
+    
+
+   /**
+   * Create program.
+   * @method
+   * @name create
+   * @param {Object} req - requested data.
+   * @returns {JSON} - created program document.
+   */
+
+    async create(req) {
+      return new Promise(async (resolve, reject) => {
+        try {
+          let programCreationData = await programsHelper.create(
+            req.body,
+            req.userDetails.userId,
```

Comment:
@Prajwal17Tunerlabs do you get userId from req.userDetails ? In authenticator you have added userId in userInformation